plotly/tools.py

```diff
@@ -2274,6 +2310,160 @@ def create_candlestick(open, high, low, close,
 
         return dict(data=data, layout=layout)
 
+    @staticmethod
+    def create_distplot(hist_data, group_labels,
+                        bin_size=1., curve_type='kde',
+                        show_hist=True, show_curve=True,
+                        show_rug=True, **kwargs):
```

Comment:
it doesn't look like `kwargs` is used below - If so, I'd remove it from the call signature and let the user modify the traces as a return argument